src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java

```diff
@@ -136,6 +169,60 @@ public void identify(IdentifyPayload identify) {
     }
   }
 
+  private void incrementAndSetTraits(Traits traits) {
+    Identify identify = new Identify();
+
+    for (Map.Entry<String, Object> entry : traits.entrySet()) {
+      String key = entry.getKey();
+      Object value = entry.getValue();
+      if (traitsToIncrement.contains(key)) {
+        if (value instanceof Double) {
```

Comment:
it might be better to split this block into it's own function, say `addTrait(key, value)`